servers/dsub/jobs/controllers/jobs_controller.py

```diff
@@ -73,28 +75,26 @@ def get_job(id):
     Returns:
         JobMetadataResponse: Response containing relevant metadata
     """
-    proj_id, job_id, task_id = job_ids.api_to_dsub(id, _provider_type())
-    provider = providers.get_provider(_provider_type(), proj_id, _auth_token())
+    project, job, task, attempt = job_ids.api_to_dsub(id, _provider_type())
+    provider = providers.get_provider(_provider_type(), project, _auth_token())
```

Comment:
In the `dsub` code base, we have tried to keep to the nomenclature that a `job` or `task` is an object of some sort (typically a dict), whereas `job_id` or `task_id` is used for the actual ID. Is this intentionally different here?

Reply:
Good point, thinking about it more I agree its better to leave as `job_id` and `task_id`. Was just trying to shorten the line so it didn't wrap, fixed.